File: app/src/main/java/com/dertyp7214/qrcodepopup/MainActivity.java
package com.dertyp7214.qrcodepopup;

import android.graphics.BitmapFactory;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.dertyp7214.qrcodedialog.components.QRCodeDialog;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.textView).setOnClickListener(v -> {
            QRCodeDialog qrCodeDialog = new QRCodeDialog(this);
            qrCodeDialog.customImageTint(
                    BitmapFactory.decodeResource(getResources(), R.drawable.ic_launcher));
            qrCodeDialog.show("Seeeeehhhhhhr langer text LOHL!!!");
        });
    }
}

File: app/src/main/java/com/dertyp7214/qrcodepopup/MainActivity.kt
package com.dertyp7214.qrcodepopup

import android.graphics.BitmapFactory
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.dertyp7214.qrcodedialog.components.QRCodeDialog

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<View>(R.id.textView).setOnClickListener {
            val qrCodeDialog = QRCodeDialog(this)
            qrCodeDialog.customImageTint(
                    BitmapFactory.decodeResource(resources, R.drawable.icon))
            qrCodeDialog.show(getString(R.string.long_string))
        }
    }
}
